src/content/docs/ai-observability/advanced-customization/drop-sensitive-data.mdx

```diff
@@ -0,0 +1,306 @@
+* In the case of `LlmTool`, you need to create queries to account for two attributes `input` and `output`. For example:
+
+  ```sql
+      SELECT output FROM LlmTool WHERE output RLIKE ^([0-9]{1,3})\.([0-9]{1,3})\.([0-9]{1,3})\.([0-9]{1,3})$
+      ```
+  ```sql
+    SELECT input FROM LlmTool WHERE input RLIKE ^([0-9]{1,3})\.([0-9]{1,3})\.([0-9]{1,3})\.([0-9]{1,3})$
+    ```
+
+### Regex
+
+Since the agent's default behavior is to capture all parts of event data before sending it to NRDB, you need to direct the agent to match sensitive information with regex. By targeting an attribute with regex, you can still capture the event itself without storing sensitive information in our databases. 
+
+Refer to the regex below to start building your first queries:
+
+<CollapserGroup>
+  <Collapser
+    id="cahcn"
+    title="Canada health card number"
+  >
+    **Expression:**
+    ```
+    (\d{10})
+    ```
+  </Collapser>
+
+  <Collapser
+    id="caphin"
+    title="Canada personal health/social insurance number (PHIN/SIN)"
+  >
+    **Expression:**
+    ```
+    (\d{3}[-\s\.]?\d{3}[-\s\.]?\d{3})
+    ```
+  </Collapser>
+
+  <Collapser
+    id="email"
+    title="Email address"
+  >
+    **Expression:**
+    ```
+    ([a-zA-Z0-9!#$'*+?^_`{|}~.-]+(?:@|%40)(?:[a-zA-Z0-9-]+\.)+[a-zA-Z0-9-]+)
+    ```
+  </Collapser>
+
+  <Collapser
+    id="indiapanid"
+    title="India PAN ID"
+  >
+    **Expression:**
+    ```
+    ^([a-zA-Z]){5}([0-9]){4}([a-zA-Z]){1}?$
+    ```
+  </Collapser>
+
+  <Collapser
+    id="indiaaadhaar"
+    title="India AADHAAR ID"
+  >
+    **Expression:**
+    ```
+    ^([2-9]{1}[0-9]{3}\s\d{4}\s\d{4})$
+    ```
+  </Collapser>
+
+  <Collapser
+    id="ipv4"
+    title="IP address (ipv4)"
+  >
+    **Expression:**
+    ```
+    ^([0-9]{1,3})\.([0-9]{1,3})\.([0-9]{1,3})\.([0-9]{1,3})$
+    ```
+  </Collapser>
+
+  <Collapser
+    id="japanid"
+    title="Japan national identity number (My number)"
+  >
+    **Expression:**
+    
+    ```
+   (d{4}\sd{4}\sd{4})
+    ```
+  </Collapser>
+
+  <Collapser
+    id="spainnid"
+    title="Spain National ID (NIE/DNI/NIF)"
+  >
+    **Expression:**
+    ```
+    ([a-zA-Z]?[-\s]?\d{7,8}[-\s]?[a-zA-Z])
+    ```
+  </Collapser>
+
+  <Collapser
+    id="ssn"
+    title="US social security number"
+  >
+    **Expression:**
+    ```
+    (\d{3}[-\s\.]?\d{2}[-\s\.]?\d{4})
+    ```
+  </Collapser>
+
+  <Collapser
+    id="uknino"
+    title="UK national insurance number (NINO)"
+  >
+    **Expression:**
+    ```
+    ([a-zA-Z]{2}[-\s]?\d{2}[-\s]?\d{2}[-\s]?\d{2}[-\s]?[a-dA-D])
+    ```
+  </Collapser>
+
+  <Collapser
+    id="usstreetaddress"
+    title="US street address"
+  >
+    **Expression:**
+    ```
+    /\d{1,}(\s{1}\w{1,})(\s{1}?\w{1,})/g
+    ```
+  </Collapser>
+
+  <Collapser
+    id="usphone"
+    title="US phone number"
+  >
+    **Expression:**
+    ```
+    (^[\+]?[1]?[\W]?[(]?[0-9]{3}[)]?[-\s\.]?[0-9]{3}[-\s\.]?[0-9]{4})
```

Comment:
I *think* we will need to remove the `^` character (second character), but I have not tested it. This is similar to the above in that it will only work if the string starts with the phone number but not if the phone number is embedded in the middle of a larger string.